index.js

```diff
@@ -5,15 +5,28 @@ const isCI = require('is-ci');
 const path = require('path');
 const fs = require('fs');
 const spawnSync = require('child_process').spawnSync;
+const execSync = require('child_process').execSync;
 const npmMergeDriver = require.resolve('npm-merge-driver');
-const findRoot = require('find-root');
+
+const findGitRoot = function(workingDir) {
+  try {
+    // Get the git root directory and strip any newlines
+    return execSync('git rev-parse --show-toplevel', {cwd: workingDir}).toString().replace(/(\r\n|\n|\r)/gm, '');
+  } catch (error) {
+    // the above git command will throw an error if not inside a working tree
+    // Most likely a localized version of the following string:
+    // "fatal: not a git repository (or any of the parent directories): .git"
+    console.log(error.message);
+    process.exit(0);
+  }
+};
 
 let rootDir;
 
 if (process.env.INIT_CWD) {
```

Comment:
We should add the `console.log('npm-merge-driver-install');` above this if statement now that finding the root directory can log an error. That way users will know where the error is coming from.